### PR TITLE
Add a Duplicate feature + more

### PR DESCRIPTION
**Pull request by BachToTheFuture**
New features + changes: fixes issues #12 and #25
* a shortcut key change: Delete task used to be "D" but now it is the "X" key.
* Duplicate option is in the menu. The shortcut key is the "D" key.
* select a task in the list and go to menu and press duplicate. A duplicate will be created, with the exact same time, status, except that it is a separate list item and you can control it separately from the original task.
* Also changed time formats. The time is more readable.

### CODE REVIEW

**Comment by humdingerb** [comment]
I have to wonder about the usefulness of duplicating a task. What's the usecase for having a copy of a task that includes the time? Mind you, I'm unable to find a usecase for a copy that starts with a reset time either...

Maybe to save typing when the title is long? That may be it... dunno...

File: src/TimeTrackerWindow.cpp [hunk 1]
```diff
@@ -142,6 +145,20 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
 		}
 		break;
 	}
+	case MENU_APP_DUPLICATE_TASK:
+	{
+		int32 selection = m_ListView->CurrentSelection();
+		if (selection >= 0) {
+			TaskListItem*	temp = (TaskListItem*)m_ListView->ItemAt(selection);
```
Comment (humdingerb) [inline src/TimeTrackerWindow.cpp]: A space instead of the tab

File: src/TimeTrackerWindow.cpp [hunk 2]
```diff
@@ -142,6 +145,20 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
 		}
 		break;
 	}
+	case MENU_APP_DUPLICATE_TASK:
```
Comment (humdingerb) [inline src/TimeTrackerWindow.cpp:148]: Can you check if more than one item is selected and if so, duplicate all of them?

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: src/TimeTrackerWindow.cpp (hunk 2 above)
Comment (BachToTheFuture) [reply src/TimeTrackerWindow.cpp:148]: Done! 😄 @humdingerb

Comment (humdingerb) [reply src/TimeTrackerWindow.cpp:148]: Now item 0 will always get duplicated, too. :)

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: src/TimeTrackerWindow.cpp (hunk 2 above)
Comment (BachToTheFuture) [reply src/TimeTrackerWindow.cpp:148]: Hehe. I'll fix that 👍

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**